Data/Sys/Shaders/sharp_bilinear.glsl

```diff
@@ -0,0 +1,49 @@
+// Based on https://github.com/libretro/slang-shaders/blob/master/interpolation/shaders/sharp-bilinear.slang
+// by Themaister, Public Domain license
+// Does a bilinear stretch, with a preapplied Nx nearest-neighbor scale, giving a
+// sharper image than plain bilinear.
+
+/*
+[configuration]
+
+OptionRangeFloat
+GUIName = Prescale Factor (set to 0 for automatic)
+OptionName = PRESCALE_FACTOR
+MinValue = 0.0
+MaxValue = 16.0
+StepAmount = 1.0
+DefaultValue = 0.0
+
+[/configuration]
+*/
```

Comment:
```suggestion
/*
[configuration]
[OptionRangeFloat]
GUIName = Prescale_Factor
OptionName = PRESCALE
MinValue = 0.0
MaxValue = 16.0
StepAmount = 1.0
DefaultValue = 0.0
[/configuration]
```
I just figured this out. You then call it with PRESCALE, but it cannot be a part of a condition.